packages/webpack-config/jest.config.js

```diff
@@ -0,0 +1,187 @@
+// For a detailed explanation regarding each configuration property, visit:
```

Comment:
i think all the jest-related stuff should be located in the root dir, to be later extended by the packages

Reply:
Moved the jest-config to root